### PR TITLE
implemented  price filtering for (TBA & 0)

### PR DESCRIPTION
**Pull request by mhyemon95**

### CODE REVIEW

File: home/ryans.py [hunk 1]
```diff
@@ -46,7 +46,9 @@ def scrape_ryans(query):
             price = driver.find_element(By.XPATH, f'//*[@id="search-box-html"]/div[4]/div/div/div[{item_id}]/div[1]/div[2]/p[3]')
             image = driver.find_element(By.XPATH, f'//*[@id="search-box-html"]/div[4]/div/div/div[{item_id}]/div[1]/div[1]/a/img').get_attribute('src')
             link = title.get_attribute('href')
-
+            #print("ryans " + price.text)
```
Comment (mominul) [inline home/ryans.py]: These two comments are unneeded now, so can you remove them?

**Review by mominul — CHANGES_REQUESTED** [review]

File: home/startech.py [hunk 2]
```diff
@@ -43,7 +43,9 @@ def scrape_startech(query):
             price = driver.find_element(By.XPATH, f'//*[@id="content"]/div[2]/div[{item_id}]/div/div[2]/div[2]/span')
             image = driver.find_element(By.XPATH, f'//*[@id="content"]/div[2]/div[{item_id}]/div/div[1]/a/img').get_attribute('src')
             link = title.get_attribute('href')
-
+            #print("startech " + price.text)
```
Comment (mominul) [inline home/startech.py]: And this

**Review by mominul — CHANGES_REQUESTED** [review]

File: home/ryans.py (hunk 1 above)
Comment (mhyemon95) [reply home/ryans.py]: Ok. I will do this.

File: home/startech.py (hunk 2 above)
Comment (mhyemon95) [reply home/startech.py]: Yeah sure.

**Review by mominul — APPROVED** [review]
Thanks!

**Pull request merged**